examples/BulkChangeReplyEmail/Ballerina.toml

```
@@ -0,0 +1,14 @@
+[package]
+org = "wso2"
+name = "BulkChangeReplyEmail"
```

This can be changed to:
```suggestion
name = "bulk_change_email"
```